### PR TITLE
[BUGFIX] Center link arrow

### PR DESCRIPTION
**Pull request by mwarning**
The link arrow is not properly centered. Not sure if this should go into some CSS file..

### CODE REVIEW

File: lib/infobox/link.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ define(['helper'], function (helper) {
 
     var arrow = document.createElement('span');
     arrow.classList.add('ion-arrow-right-c');
+    arrow.style.verticalAlign = 'middle';
```
Comment (xf-) [inline lib/infobox/link.js:24]: No style via JS. I think the problem was different rendering on different platforms and with different fonts.

**Review by xf- — CHANGES_REQUESTED** [review]

File: lib/infobox/link.js (hunk 1 above)
Comment (genofire) [reply lib/infobox/link.js:24]: 👍 for @xf- comment

**Comment by mwarning** [comment]
.. where has the CSS property to be placed then? I would argue that the arrow should be in the vertical middle independently of the font.

**Comment by xf-** [comment]
fonts are really complex - icon should normally aligned on baseline.

**Comment by mwarning** [comment]
I use Firefox ESR 52.3.0 (64 bit).
This is how the arrow looks like https://imgur.com/a/UGByW

**Comment by mwarning** [comment]
Would it be acceptable to add the property to a CSS file?

**Comment by xf-** [comment]
Yes, but it not depends so much on browser direct as on FreeType on Linux/Mac OS and Windows font rendering vs different baseline stuff in different fonts.

(I don't like that view - long node names always break into a new line)

**Comment by mwarning** [comment]
@xf-  a: white-space: nowrap; might do the trick to avoid line breaking in link texts.
As for the arrow, where do I have to add the css? Probably somewhere in scss/ ...

**Comment by xf-** [comment]
We need a break. On small devices it is not avoidable.
My idea was more like 

```
Form: Node 1
To: Node 2
```
and remove the arrow. Currently locally i replaced the arrow with a `-` because it is used in all other views.
It is very likely the arrow will be replaced

**Comment by xf-** [comment]
Current development
image
What do you think?

**Comment by mwarning** [comment]
@xf- looks fine. In this case we could then close this ticket.

**Comment by xf-** [comment]
If you want to close it OK.
We have much more space in sidebar and for example the router versions need a lot fo space.
I tried to use a table with from and to, but it looked horrible.

**Comment by mwarning** [comment]
Maybe put the link information (distance, link connectivity, ..) at the top of the infobox and two short node descriptions (name, model, node connectivity ...) as separate sections below. That would avoid a line with "<model> - <model>".

**Pull request closed without merging**